README.md

```diff
@@ -1 +1,166 @@
-# portfolio-desktop-version
\ No newline at end of file
+# Jose RM - Projects Portfolio
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- TABLE OF CONTENTS -->
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] As part of the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/5_deploy.md), kindly include this `Live Demo` section in your ReadMe content, and add your live demo link i.e. a link to your deployed portfolio so that your page will be easily accessible to users or visitors of your GitHub page who might want to check your awesome work. 😉 

![image](https://user-images.githubusercontent.com/99715640/215660657-1af4e463-b869-4da5-9d4f-3bdc91814131.png)